Gordon360/Models/ViewModels/RecIM/TeamRecordViewModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Gordon360.Models.CCT;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class TeamRecordViewModel
+    {
+        public int ID { get; set; }
+        public string Name { get; set; }
+        public int Win { get; set; }  
+        public int Loss { get; set; }
+        public int Tie { get; set; }
```

Comment:
```suggestion
        public int WinCount { get; set; }  
        public int LossCount { get; set; }
        public int TieCount { get; set; }
```
Adding count to these fields will clarify exactly what the values are. Otherwise, you have to infer that they are counters.

Reply:
Yes, that thought makes sense. Data sometimes has to be more descriptive in nomenclature than the fields on a UI.